src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -284,7 +284,6 @@ public String getQuestionSpecificEditFormHtml(int questionNumber) {
                 Slots.RECIPIENT_DISPLAY, distributeToRecipients ? "" : "style=\"display:none\"",
                 Slots.PER_OPTION_CHECKED, !distributeToRecipients && pointsPerOption ? "checked" : "",
                 Slots.PER_RECIPIENT_CHECKED, distributeToRecipients && pointsPerOption ? "checked" : "",
-                Slots.OPTIONRECIPIENTDISPLAYNAME, distributeToRecipients ? "recipient" : "option",
```

Comment:
It is the duplicate of `Slots.OPTION_RECIPIENT_DISPLAY_NAME`. Not meant for this PR. Should I do this in different PR ?